jest.config.ts

```diff
@@ -24,6 +24,15 @@ const config: Config = {
   // Indicates which provider should be used to instrument code for coverage
   coverageProvider: 'v8',
 
+  coverageThreshold: {
+    global: {
+      branches: 100,
+      functions: 100,
+      lines: 100,
+      statements: -10,
```

Comment:
Typo?

Reply:
No, it specifies that f there are more than 10 lines not covered, the test should fail.